support/sys-statvfs.c

```diff
@@ -51,7 +51,15 @@ Mono_Posix_ToStatvfs (void *_from, struct Mono_Posix_Statvfs *to)
 	to->f_files   = from->f_files;
 	to->f_ffree   = from->f_ffree;
 	to->f_favail  = from->f_favail;
+	/*
+         * On AIX with -D_ALL_SOURCE, fsid_t is a struct
```

Comment:
I've run into stuff like this on VMS, the stat.ino. Is the thing always the same size and offset, can you cast it to the right size integer pointer and read that?

Reply:
Also, is this autoconfable?